bool wait_until_published
---
bool success